documentation/api/processes-get.md

```diff
@@ -0,0 +1,78 @@
+# Processes - Get
+
+Gets detailed information about a specified process.
+
+```http
+GET https://localhost:52323/processes/{pid}
+```
+
+or 
+
+```http
+GET https://localhost:52323/processes/{uid}
+```
+
+## URI Parameters
+
+| Name | In | Required | Type | Description |
+|---|---|---|---|---|
+| `pid` | path | true | int | The ID of the process. |
+| `uid` | path | true | guid | A value that uniquely identifies a runtime instance within a process. |
+
+See [ProcessIdentifier](processes-list.md#ProcessIdentifier) for more details about these parameters.
+
+Either `pid` or `uid` are required, but not both.
+
+## Responses
+
+| Name | Type | Description |
+|---|---|---|
+| 200 OK | [ProcessInfo](#ProcessInfo) | The process information |
+| 400 Bad Request | ValidationProblemDetails |  |
+| 401 Unauthorized | | Authentication is required to complete the request. See [Authentication](authentication.md) for further information. |
+
+## Examples
+
+### Sample Request
```

Comment:
Should sample requests contain an auth header?

Reply:
I think that's reasonable.